app/models/trends/tag_filter.rb

```diff
@@ -14,7 +14,7 @@ def initialize(params)
 
   def results
     scope = if params[:status] == 'pending_review'
-              Tag.unscoped
+              Tag.unscoped.order(created_at: :desc)
```

Comment:
I think we should order by `id: desc` instead, this would result in the same order, but using the index on the primary key, instead of doing a potentially very expensive operation (we do not have an index on `created_at`)

```suggestion
              Tag.unscoped.order(id: :desc)
```

Reply:
Sure, that works too!